pkgs/leak_tracker/README.md

```diff
@@ -1,6 +1,18 @@
 [![pub package](https://img.shields.io/pub/v/leak_tracker.svg)](https://pub.dev/packages/leak_tracker)
 [![package publisher](https://img.shields.io/pub/publisher/leak_tracker.svg)](https://pub.dev/packages/leak_tracker/publisher)
 
-Coming soon!
+Coming soon! See https://github.com/flutter/devtools/issues/3951 and https://github.com/flutter/devtools/issues/5606.
 
-See full documentation at https://github.com/dart-lang/leak_tracker.
+The text below is under construction.
+
+This is a framework for detecting memory issues in Dart and Flutter applications.
+
+It enables:
```

Comment:
I think it would be useful to re-organize this into a few sections and populate those a bit more. Off-hand, I'd suggest `## What's this?`, `## Usage` (containing an example), and `## Additional documentation`.